templates/new/config/target.exs

```diff
@@ -35,16 +38,32 @@ keys =
   ]
   |> Enum.filter(&File.exists?/1)
 
-if keys == [],
-  do:
-    Mix.raise("""
-    No SSH public keys found in ~/.ssh. An ssh authorized key is needed to
-    log into the Nerves device and update firmware on it using ssh.
-    See your project's config.exs for this error message.
-    """)
+if keys == [] do
```

Comment:
I think that having a way of creating a username/password config is a good idea, but I don't think it should depend on whether you have ssh keys or not. 

Rather than delay the 0.4 part of this, would you mind splitting the PR up into a `nerves_pack v0.4` update and a user/pass update?

Reply:
Removed.